app/uploaders/place_photo_uploader.rb

```diff
@@ -1,17 +1,22 @@
 class PlacePhotoUploader < CarrierWave::Uploader::Base
+  if Rails.env.test?
+    storage :file
+  else
+    storage :fog
+  end
   # Include RMagick or MiniMagick support:
   # include CarrierWave::RMagick
   # include CarrierWave::MiniMagick
 
   # Choose what kind of storage to use for this uploader:
-  storage :file
-  # storage :fog
-
-  # Override the directory where uploaded files will be stored.
-  # This is a sensible default for uploaders that are meant to be mounted:
-  def store_dir
```

Comment:
I would expect you to still need this function, since you're using `storage :file` if in a test env